marsha/llm.py

```diff
@@ -10,7 +10,7 @@
 
 from pylama.main import parse_options, check_paths, DEFAULT_FORMAT
 
-from marsha.parse import validate_first_stage_markdown, validate_second_stage_markdown, write_files_from_markdown, format_marsha_for_llm, extract_func_name, extract_type_name
```

Comment:
did we check if this was not in use anywhere else? otherwise, we should delete it from parse.py

Reply:
Yeah, it's used in `base.py`

```sh
damocles@elack:~/alantech/marsha/marsha(lint)$ git grep extract_type_name
base.py:from marsha.parse import extract_functions_and_types, extract_type_name, write_files_from_markdown, is_defined_from_file, extract_type_filename
base.py:        type_name = extract_type_name(raw_type)
parse.py:def extract_type_name(type):
```